install/install.sh

```diff
@@ -80,11 +80,12 @@ else
        read -e -p "Invalid password, please retry: " -i "" db_user_password
```

Comment:
This case (existing user) still will throw warnings, please add some code to mitigate it here as well.